api/v1alpha1/common.go

```diff
@@ -7,13 +23,14 @@ var (
 	DisplayNameAnnotationKey = GroupVersion.Group + "/displayname"
 )
 
-// ComponetVersions Indicates the fields required for a specific version of Component.
-type ComponetVersions struct {
+// ComponentVersions Indicates the fields required for a specific version of Component.
+type ComponentVersions struct {
```

Comment:
Just thought of this, should we use `ComponentVersion` here instead of its plural form `ComponentVersions`?